editors/sced/README.md

```diff
@@ -48,7 +48,7 @@ Place wrapper-scripts for sclang and scsynth into `/usr/local/bin`.
 
 ```
 #!/bin/sh
-cd /full/path/to/SuperCollider.app/Contents/Resources
+cd /full/path/to/SuperCollider.app/Contents/MacOS
```

Comment:
@bagong this is right now. it was wrong. right ?  I just happened to notice it while I was in there.